beacon-chain/db/filesystem/blob.go

```diff
@@ -238,15 +236,8 @@ func (p blobNamer) path() string {
 // Prune prunes blobs in the base directory based on the retention epoch.
 // It deletes blobs older than currentEpoch - (retentionEpochs+bufferEpochs).
 // This is so that we keep a slight buffer and blobs are deleted after n+2 epochs.
-func (bs *BlobStorage) Prune(currentSlot primitives.Slot) error {
+func (bs *BlobStorage) Prune(pruneBeforeSlot primitives.Slot) error {
```

Comment:
nitpick: we know the type of this variable is a slot, `pruneBefore` would be succinct and obvious.